install/gcp/installer/forseti_server_installer.py

```diff
@@ -272,6 +272,8 @@ def get_deployment_values(self):
             'FORSETI_BUCKET': bucket_name[len('gs://'):],
             'BUCKET_LOCATION': self.config.bucket_location,
             'GCP_SERVER_SERVICE_ACCOUNT': self.gcp_service_acct_email,
+            'FORSETI_SERVER_REGION': self.config.cloudsql_region,
```

Comment:
This is an improvement, and will be accepted.  But the inherent oddity is still present... that is we are re-using a value for the cloudsql for the VM.  Either that flag should be renamed `--cloudsql-region` to just `--deployment-region`, or a new flag for the VM should be added `--vm_region`.  I realized this is a bigger change, so thoughts?